packages/harmony/package.json

```diff
@@ -81,15 +90,17 @@
     "typescript-plugin-css-modules": "3.4.0",
     "webpack": "4.46.0"
   },
-  "dependencies": {
-    "@emotion/react": "11.11.1",
-    "@emotion/styled": "11.11.0",
-    "@radix-ui/react-slot": "1.0.2",
-    "classnames": "2.2.6",
-    "react-lottie": "1.2.3",
-    "react-merge-refs": "2.0.1",
-    "react-perfect-scrollbar": "1.5.8",
-    "react-use": "15.3.8",
-    "react-use-measure": "2.1.1"
+  "peerDependencies": {
```

Comment:
Do these actually need to be peer dependencies? Wouldn't this require any consumer to manually add each of these to their package.json?

If we leave them as regular deps, the bundle will still be small because they are all externalized. But the deps will automatically installed for any consumer

Reply:
so yes they will be externalized but still show up as separate blobs in the bundles i think? the peers is the way i believe to ensure that harmony deps get determined and set by user?

Reply:
Just coming from a library consumer perspective, it would be pretty inconvenient to have to manually add 10+ packages to my own package.json to get harmony to work. Ideally it would just be `npm add @audius/harmony`

If the semver ranges are loose enough npm should resolve to a single version and not install a separate version specifically for harmony. And the consumer can always use npm overrides or yarn resolutions if they do want to force a specific version

Reply:
Yeah I totally understand that point, but for react and emotion, in order for those to not explode your dep size you would need to match your versions exactly. I'm down to figure out the best path for this long term, but to prevent harmony bundle size issues for now we cool with starting like this?